products/ha_memgraph/v0.15.0/how_to_guides/configure-and-run-ha-memgraph.md

```diff
@@ -0,0 +1,117 @@
+## How to Configure and Run HA Memgraph?
+
+High availability (HA) is a characteristic of a system which aims to ensure a
+certain level of operational performance for a higher-than-normal period.
+Although there are multiple ways to design highly-available systems, HA Memgraph
+strives to achieve HA by elimination of single points of failure. In essence,
+this implies adding redundancy to the system so that a failure of a component
+does not imply the failure of the entire system. To ensure this, HA Memgraph
+implements the [Raft Consensus Algorithm](https://raft.github.io/). For more
+information on Raft internals as well as its integration with HA Memgraph, you
+are advised to read [this article](../concepts/high-availability.md).
+
+### Cluster Setup
+
+In order to successfully use HA Memgraph, you need to run the
+`memgraph_ha` binary on each machine in your cluster. We recommend running this
+feature on an odd-sized cluster. The most commonly used cluster size is 3 and
+all examples throughout this guide will assume that.
+
+In order to run the `memgraph_ha` binary, we need to specify a couple of things
+in advance. Each server within the cluster should be given a unique ID value
+between 1 and `cluster_size`. Since we are dealing with a cluster of 3 machines,
+their ID values will be 1, 2 and 3.
+
+We also need to specify two `.json` files. One of them (`raft_config_file`)
+defines some constants which are internal to the Raft protocol. The other one
+(`coordination_config_file`) contains coordination info consisting of
+`server_id`, `ip_address` and `rpc_port`.
+
+Therefore, we can run the `memgraph_ha` binary on the first (`ID=1`) server by
+issuing the following command:
+
+```plaintext
+  ./memgraph_ha --server_id 1 \
+                --coordination_config_file="coordination.json" \
+                --raft_config_file="raft.json" \
+                --port 1001
```

Comment:
Ports 0 to 1023 are Well-Known Ports. Pick a bigger number.

Reply:
Since each HA memgraph instance is running on a different machine we don't need to specify the port used for bolt protocol. The default value is 7687 and I don't see a reason why don't we just use the default.